client/src/css/stylesheet.scss

```diff
@@ -35,177 +42,315 @@ body {
           }
         }
 
         #cruise {
+          position: absolute;
+          top: 30%;
+          width: 50%;
           text-align: left;
+          padding-left: 3em;
 
           #cruise-value {
-            padding-left: 0.25em;
-            font-size: 2.5em;
+            margin: 0;
+            font-size: 3em;
           }
         }
       }
-    }
-
-    #soc-wrapper {
-      width: 30%;
-      height: 40%;
-      position: absolute;
-      left: 27%;
-      top: 55%;
-      overflow: visible;
-      float: right;
 
-      #soc-icon {
-        width: 100%;
-        height: 100%;
+      #soc-wrapper {
+        width: 45%;
+        height: 50%;
+        overflow: visible;
+        display: flex;
         position: relative;
-        z-index: 1;
 
-        #battery-icon {
+        #soc-icon {
+          width: 50%;
+          height: 100%;
           position: absolute;
-          top: 65%;
-          left: 10%;
-          width: 35%;
-          height: 35%;
+          z-index: 1;
+
+          #battery-icon {
+            position: absolute;
+            width: 35%;
+            height: 30%;
+            right: -20%;
+            bottom: 10%;
+          }
         }
-      }
 
-      #soc {
-        position: absolute;
-        top: 0%;
-        left: 0%;
-        z-index: 3;
-        height: 100%;
+        #soc {
+          position: absolute;
+          z-index: 3;
+          bottom: 17%;
+          right: -10%;
+          height: 75%;
+          width: 100%;
+
+          #soc-path {
+            fill-opacity: 0;
+            stroke-linecap: "butt";
+            stroke: #11b981;
+            stroke-opacity: 1;
+            stroke-width: 2.5rem;
+          }
+
+          #soc-text {
+            display: none;
+          }
+        }
       }
     }
 
-    #graph-wrapper {
+    #readout-wrapper {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: center;
+      align-content: space-around;
+      align-items: center;
+      width: 50%;
+      height: 88%;
+      top: 12%;
+      right: 0%;
       position: absolute;
-      right: 6%;
-      top: 15%;
-      width: 45%;
-      height: 45%;
 
-      #graph {
+      #graph-wrapper {
         position: relative;
-        height: 100%;
-        width: 100%;
+        width: 88%;
+        height: 45%;
 
-        .tick-label {
-          fill: white;
-        }
+        #graph {
+          position: relative;
+          height: 100%;
+          width: 100%;
 
-        .text-line {
-          fill: white;
-        }
+          .tick-label {
+            fill: white;
+            font-size: 1em;
+          }
+
+          .text-line {
+            fill: white;
+          }
+
+          .x-axis-tick .tick-label {
+            // Force the visibility on the x axis. The end tick labels are being
+            // overridden to visiblility: hidden in the Plottable JS library
+            // explicitly and I don't want to fiddle with the Plottable src.
```

Comment:
Yeah, it's the only way you'll be able to override the CSS cascade (without actually fixing the code in plottable), since its applying the style inline using `style`.